_docs/install/openshift/index.md

```diff
@@ -18,26 +18,57 @@ type to install to make devices usable from within containers.
 
 You will need an OpenShift 3.8+ cluster with Beta APIs enabled.
 
-1. Enable the `MountPropagation` flag by appending `--feature-gates
-MountPropagation=true` to the kube-apiserver and kubelet services.
+1. Make sure your docker installation has mount propagation enabled.
+```
+# A successful run is proof that mount propagation is enabled
+docker run -it --rm -v /mnt:/mnt:shared busybox sh -c /bin/date
+```
+1. The [iptables rules]({% link _docs/install/prerequisites/firewalls.md %}) required for StorageOS.
+1. Enable the `MountPropagation` flag by appending feature gates to the api and controller (you can apply these changes using the Ansible Playbooks)
+- Add to the KubernetesMasterConfig section (/etc/origin/master/master-config.yaml):
+
+    ```
+kubernetesMasterConfig:
+  apiServerArguments:
+    feature-gates:
+    - MountPropagation=true
+  controllerArguments:
+    feature-gates:
+    - MountPropagation=true
+    ```
+
+- Add to the feature-gates to the kubelet arguments (/etc/origin/node/node-config.yaml):
+
+    ```
+kubeletArguments:
+  feature-gates:
+  - MountPropagation=true
+    ```
 
-1. If the kubelet is running in a container, add `--volume=/var/lib/storageos:/var/lib/storageos:rshared` to each
-of the kubelets. This will be [fixed](https://github.com/kubernetes/kubernetes/pull/58816) in OpenShift 3.10+.
+- **Warning:** Restarting OpenShift services can cause downtime in the cluster.
+- Restart services in the MasterNode `origin-master-api.service`, `origin-master-controllers.service` and `origin-node.service`
+- Restart service in all Nodes `origin-node.service`
 
-1. The [iptables rules]({% link
-_docs/install/prerequisites/firewalls.md %}) required for StorageOS.
 
 For OpenShift 3.7, you can [install the container directly in
 Docker]({%link _docs/install/docker/index.md %}).
 
 ## Install with Helm
 
-Firstly, [install Helm](https://blog.openshift.com/getting-started-helm-openshift).
+StorageOS container needs privileged execution permissions. Because of that it is required to add a security context constraint. 
```

Comment:
"The StorageOS container needs privileged execution permissions, so a security context constraint must be added. "